documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+|                                           | **Use Case Example 1 - Wi-Fi Congestion**<br>Wi-Fi congestion at a `Service Site` is affecting video conferencing performance, reflecting lower QbD scores and poor video quality                                                                                                                                                                                                                                                                                                                                                                                                                   |
+|                                           | **Use Case Example 2 - Network Impairments**<br>Issues in network operator's network can lead to degradation in latency, jitter, packet loss, and bitrate, reflecting lower QbD scores and poor video quality.                                                                                                                                                                                                                                                                                                                                                                                      |
+|                                           | **Sample Recommendations**<br>- speed boost<br>- traffic prioritization<br>- WMM enablement<br>- application configuration updates<br>- request service improvements                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| Technical viability                       | This API is for 3rd party application developers to consume. It assumes the network operator exposing this API has a cloud backend to receive KPIs, analyze them, and produce a QbD score. Based on thresholds for the app, the network operator can provide proactive recommendations. The 3rd party application could also use the Camara NetworkAccessManagement for remote control of a `Device` at a `Service Site` if necessary.                                                                                                                                                              |
```

Comment:
`Service Site` same comment as above.

Reply:
got it - linked to `Service Site` definition and also to `Device` definition  in the NetworkAccessManagement definnition also for consistency.